biogrid/apps/biogrid-mvp/src/app/pages/input-page/input-page.tsx

```diff
@@ -22,92 +30,128 @@ export const InputPage = () => {
   const [startDate, setStartDate] = useState(new Date());
   const [endDate, setEndDate] = useState(new Date());
   const [smallBatteryCells, smallBatteryCellInput] = useInput({
-    type: 'number',
+    default: 0,
```

Comment:
If all of these default to 0, can you automatically set that default in the `useInput()` method?

Reply:
Oh yeah, earlier there were strings and such, we may add strings in the future though